lib/ui/components/bottom_action_bar/expanded_menu_widget.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:photos/ui/components/blur_menu_item_widget.dart';
+import 'package:photos/ui/components/divider_widget.dart';
+
+class ExpandedMenuWidget extends StatelessWidget {
+  final List<BlurMenuItemWidget> items;
+  final List<int> groupingOrder;
+  const ExpandedMenuWidget({
+    required this.items,
+
+    /// To specify the grouping of items. Eg: [2,2] for 2 sections with 2 items
+    /// each, [4,1,2] for 3 sections with 4, 1 and 2 items respectively.
+    /// Make sure sum of ints in this.groupingOrder is equal to length of this.items
+    required this.groupingOrder,
```

Comment:
Why not take a `List<List<BlurMenuItemWidget>>`s so that you don't need `groupingOrder`?